package.json

```diff
@@ -36,7 +36,7 @@
     "open-hot": "builder concurrent hot open-demo",
     "docs-dev": "webpack-dev-server --port 3000 --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.dev.js --content-base docs",
     "docs-hot": "webpack-dev-server --port 3000 --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.hot.js --hot --content-base docs",
-    "docs-build-static": "webpack --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.static.js --progress",
+    "docs-build-static": "BABEL_ENV=webpack webpack --config node_modules/builder-victory-component/config/webpack/docs/webpack.config.static.js --progress",
```

Comment:
NOTE: This will break on windows if we intend to support windows. (maybe not for the docs, but victory components should build on windows)